config/containers/logging/dual-logging.md

```diff
@@ -0,0 +1,114 @@
+---
+description: Learn how to read container logs locally when using a third party logging solution.
+keywords: docker, logging, driver
+title: Reading container logs locally with remote logging drivers
```

Comment:
"Reading container logs locally with remote logging drivers" should be 

"Using `docker logs` to read container logs for remote logging drivers"

Reply:
Got it.